CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/DataAccessViewTopComponent.java

```diff
@@ -144,19 +155,49 @@ protected void componentShowing() {
 
     @Override
     protected void handleNewGraph(final Graph graph) {
-        System.setProperty("dav.graph.ready", Boolean.FALSE.toString());
         if (needsUpdate() && getDataAccessPane() != null) {
             getDataAccessPane().update(graph);
-            Platform.runLater(() ->
+            Platform.runLater(() -> 
                 DataAccessUtilities.loadDataAccessState(getDataAccessPane(), graph)
             );
         }
-        Platform.runLater(() ->
-                // nested runLater so it runs after all the other triggered processes for opening a graph have been run
-                Platform.runLater(() -> System.setProperty("dav.graph.ready", Boolean.TRUE.toString()))
-        );
     }
-    
+
+    /**
+     * Add or remove all quality control auto vetter listeners. These listeners
+     * are tied to DefaultQualityControlAutoButton buttons found nested within
+     * the button toolbar.
+     *
+     * @param add Should quality control auto vetter listeners be added (true)
+     * or removed (false).
+     */
+    private void ManageQualityControlListeners(boolean add) {
```

Comment:
The method/function names should begin with a lower case character.
Rename this to: manageQualityControlListeners

Reply:
Done